README.md

```diff
@@ -104,7 +103,7 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+You will receive error messages because the above code does not contain the proper HTML document format structure.  Additionally, you may also receive an error because the erb tags seem to set off errors in HTML validators and the atom linter plug-in.  As long as all the errors are related to HTML document type and the erb tags having "unescaped <", you should be okay.
 ```
```

Comment:
:+1: It is the ERB that throws the validator off. A solution, however, is to copy the (ERB-less) source code from the browser and then validate that.